scss/core/_vars.scss

```diff
@@ -115,3 +121,15 @@ $upsell-padding-y: calc-em(10);
 $upsell-padding-x: calc-em(15);
 $upsell-padding-y-lg: calc-em(10);
 $upsell-padding-x-lg: calc-em(10);
+
+// Breadcrumb
+$breadcrumb-bg: white;
+$breadcrumb-divider-color: $gray-900;
+$breadcrumb-active-color: $gray-900;
```

Comment:
we say that we must use body-color

Reply:
you are still using gray-900

Reply:
not sure why you are still seeing gray-900, i have already updated on the new commit for that

Reply:
updated, looks i didn't save the changes :(